bucket/eagleget-portable.json

```diff
@@ -8,15 +8,12 @@
     "version": "2.1.5.10",
     "url": "http://dl.eagleget.com/latest/EagleGetProtable_2.1.5.10.zip",
     "hash": "sha1:d81b94fffae7ad5594844706ae59b5021256c680",
-    "extract_dir": "/",
     "shortcuts": [
         [
             "EagleGet.exe",
             "EagleGet"
         ]
     ],
-    "pre_install": "New-Item -ItemType Directory -Force -Path $dir\\EGDownloads | Out-Null",
-    "##": "EGDownloads directory will create after the first run.",
     "persist": [
         "EagleGet\\configs",
```

Comment:
Does this persist work? I had some issues with nested folders persist.

Reply:
It works in this APP.